tests/foreman/api/test_repository.py

```diff
@@ -34,8 +34,8 @@
     VALID_GPG_KEY_BETA_FILE,
     VALID_GPG_KEY_FILE,
     DOWNLOAD_POLICIES,
-    REPO_TYPE
-)
+    REPO_TYPE,
+    FAKE_1_PUPPET_REPO)
```

Comment:
Closing bracket should have been placed in new line, trailing comma is missing and basically constants are not sorted alphabetically... But as it's cherry-pick we should merge it as-is to avoid conflicts between branches later. And anyway it's a nitpick and nothing serious